spec/mail/mailbox.mdown

```diff
@@ -47,6 +68,23 @@ A **Mailbox** object has the following properties:
   The user may rename the mailbox or make it a child of another mailbox. This property is server-set.
 - **mayDelete**: `Boolean`
   The user may delete the mailbox itself. This property is server-set.
+- **namespace**: `Namespace`
```

Comment:
This property doesn't make sense. An Account is the namespace. In JMAP, if you share content with another user, that will appear as an extra account that the other user now has access to (although they may not see all of the same content when accessing that account if only part of it is shared with them).

Reply:
True.

We had some discussion internally about the notion of account.

It could very well be dropped as part of this contribution (and be concidered as an extension)

Reply:
I don't understand how having access to another Account solves that. Maybe it's just a wording issue, but in my understanding, sharing an account is one thing (user A can access account B), sharing mailboxes is another thing (account A containing mailboxes also contained in account B). @neilj Do you suggest that when user A share a mailbox to user B it implicitly creates a new Account accessible to user B ? Does that mean that user B has accounts managed by other users ?

Reply:
Whether you share just some mailboxes or everything from user A with user B, this would still appear in JMAP as a second account (that of user A) that user B now has access to. It's just the data returned when querying it would be different (so getMailboxes may return a smaller set of mailboxes etc.).